packages/eslint-config-eslint/default.yml

```diff
@@ -278,6 +293,52 @@ rules:
     template-curly-spacing: ["error", "never"]
     template-tag-spacing: "error"
     unicode-bom: "error"
+
+    # Disable for now
```

Comment:
I'm confused how this is grouped with the blank lines. Can we just keep the unicorn rules together and alphabetically sorted?

Reply:
I've added some comments (and sorted). Let me know if there is anything still unclear. I think it is useful to have the groupings though to denote how deviations were made from the recommended config